pyScienceMode2/sciencemode.py

```diff
@@ -7,7 +7,7 @@
 import time
 import threading
 from pyScienceMode2.utils import *
-from .acks import motomed_error_ack
+from .acks import *
```

Comment:
Try to not import all but import the specific class instead.

Reply:
Done.